src/components/experiment-tracking/parallel-coordinates/parallel-coordinates.js

```diff
@@ -236,6 +240,14 @@ export const ParallelCoordinates = ({
                     (value, index) => index === metricIndex && value
                   );
 
+                const xScaleMetricName = isNaN(xScale(metricName))
+                  ? 0
+                  : xScale(metricName);
+
+                const yScaleMetricName = isNaN(yScales[metricName](value))
+                  ? 0
+                  : yScales[metricName](value);
```

Comment:
I think these are tick values and not metric names aren't they?

```suggestion
                const xScaleTickValue = isNaN(xScale(metricName))
                  ? 0
                  : xScale(metricName);

                const yScaleTickValue = isNaN(yScales[metricName](value))
                  ? 0
                  : yScales[metricName](value);
```

Reply:
ah yeah good spot, thank you